### PR TITLE
Xenos keep numeric age on evolve/upgrade

### PR DESCRIPTION
**Pull request by QualityVan**
## About The Pull Request
Age is cumulative across upgrades. Before you'd need 60 for t1 young and 120 for mature, now 60 and 180 but without resetting. This value also gets carried across evolutions - you'll keep your total age going from drone to hivelord and upgrade to whatever tier you'd reach, instead of flat downgrading one tier like current. Same for devolving.

## Why It's Good For The Game
Consistency, avoids abuse/annoyances about hitting specific breakpoints before evolving.

## Changelog

:cl:
refactor: Numeric age is cumulative across xeno upgrades instead of resetting
balance: Xenos upgrade maturity based on accumulated age on evolution instead of old caste maturity level
balance: Xenos continue to gain age once they're at maximum maturity tier
/:cl:

### CODE REVIEW

**Comment by Dgeen** [comment]
Direct nerf is what Xenos need now, speedy merdge

**Comment by Zambie-Tron** [comment]
Why is this even being considered? nuke this shit. shadow nerf how is it a **buff** explain. Mandatory playtime posting for xenos-related PRs when.

**Comment by Autisem** [comment]
Can you test if this breaks the xeno pannel?(admin tool)

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -797,19 +797,19 @@ GLOBAL_LIST_INIT(human_body_parts, list(BODY_ZONE_HEAD,
...
+#define TIER_TWO_ELDER_THRESHOLD 840
+#define TIER_TWO_ANCIENT_THRESHOLD 1080
 
 #define TIER_THREE_YOUNG_THRESHOLD 250
-#define TIER_THREE_MATURE_THRESHOLD 500
-#define TIER_THREE_ELDER_THRESHOLD 1000
-#define TIER_THREE_ANCIENT_THRESHOLD 100
+#define TIER_THREE_MATURE_THRESHOLD 750
+#define TIER_THREE_ELDER_THRESHOLD 1750
+#define TIER_THREE_ANCIENT_THRESHOLD 1850
```
Comment (TiviPlus) [inline code/__DEFINES/mobs.dm:812]: Why nto change the macro to TIER_ONE_MATURE_THRESHOLD 120+TIER_ONE_YOUNG_THRESHOLD

**Review by TiviPlus — APPROVED** [review]

**Comment by Naaanii** [comment]
To the people who think it's a nerf:

image

The only difference is if you miss out a minute or two after immediately maturing, you keep the evo points when you evolve and lose out on even less progress.

**Comment by Ben10083** [comment]
People are upset regarding how it stops people from going t2 primo then evolve into t3, allowing you to become ancient faster than you would otherwise. Whether maints want to keep that or not is up to them.

**Comment by Al-1ce** [comment]
Bad.

**Pull request merged**